app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,7 +41,11 @@
     @Column(name = "operation_type")
     public String operation_type;
 
-    public static BgSendQueue nextBgJob() {
+    private static Context mContext = Home.getContext();
```

Comment:
This context was null once for me, leading to a crash.  I suspect Home had been reaped for memory and a service kicked off BgSendQueue.  Keeping static contexts around is generally considered bad form because it leads to leaking contexts.  It looks like it is probably better to use the context already being passed into the method using pebbleSync.